//package com.sincetimes.redwar.game.comnunicate.protocol;

message RseUpdateTargets
{
    optional int32 ret = 1;
}